### PR TITLE
Update imdbpie to release 5.2.0

### PR DESCRIPTION
**Pull request by p0psicles**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by duramato** [comment]
https://github.com/pymedusa/Medusa/tree/master/ext/imdbpie

**Comment by p0psicles** [comment]
It was in lib right? And I know of the deps. Look at it tomorrow. Of someone else doesnt beat me.

**Comment by duramato** [comment]
We currently have it in `ext`, you added this new version to `lib`.

File: medusa/tv/series.py [hunk 1]
```diff
@@ -1569,26 +1569,26 @@ def load_imdb_info(self):
         self.imdb_info['countries'] = self.imdb_info.get('countries', '')
         self.imdb_info['country_codes'] = self.imdb_info.get('country_codes', '')
 
-        # If the show has no year, IMDb returned something we don't want
-        if not imdb_obj or not imdb_obj.year:
-            log.debug(u'{id}: IMDb returned none or invalid info for {imdb_id}, skipping update.',
+        imdb_obj = imdb_api.get_title(self.imdb_id)
```
Comment (duramato) [inline medusa/tv/series.py]: Does it still makes sense to call it `imdb_obj`?

Comment (medariox) [reply medusa/tv/series.py]: Well, it is coming from IMDb and it is obviously an object so I don't see why not. Would imdb_dict be better? imdb_info? I can't really find a good name for it, but I'm open to suggestions.

Comment (duramato) [reply medusa/tv/series.py]: Yeah, I was thinking something like `imdb_info`

**Pull request merged**